client/cmd/job/delete.go

```diff
@@ -0,0 +1,207 @@
+package job
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/goto/salt/log"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+
+	"github.com/goto/optimus/client/cmd/internal/connection"
+	"github.com/goto/optimus/client/cmd/internal/logger"
+	"github.com/goto/optimus/client/cmd/internal/survey"
+	"github.com/goto/optimus/client/local"
+	"github.com/goto/optimus/client/local/model"
+	"github.com/goto/optimus/client/local/specio"
+	"github.com/goto/optimus/config"
+	pb "github.com/goto/optimus/protos/gotocompany/optimus/core/v1beta1"
+)
+
+const (
+	deleteTimeout = time.Minute * 5
+)
+
+type deleteCommand struct {
+	deleteSurvey    *survey.JobDeleteSurvey
+	namespaceSurvey *survey.NamespaceSurvey
+	logger          log.Logger
+
+	connection connection.Connection
+
+	configFilePath string
+	force          bool
+	cleanHistory   bool
+
+	clientConfig *config.ClientConfig
+	fs           afero.Fs
+	reader       local.SpecReader[*model.JobSpec]
+}
+
+// NewDeleteCommand initializes job delete command
+func NewDeleteCommand() *cobra.Command {
+	l := logger.NewClientLogger()
+	deleteCmd := &deleteCommand{
+		deleteSurvey:    survey.NewJobDeleteSurvey(),
+		namespaceSurvey: survey.NewNamespaceSurvey(l),
+		logger:          l,
+	}
+
+	cmd := &cobra.Command{
+		Use:     "delete <job_name>",
+		Short:   "Delete an existing job in the SERVER",
```

Comment:
we can mention this will delete from local and server too

Reply:
updated as mentioned